outputs.tf

```
@@ -8,12 +8,16 @@ output "subscription_resource_id" {
   description = "The subscription_resource_id is the Azure subscription resource id that resources have been deployed into"
 }
 
-
 output "virtual_network_resource_ids" {
   value       = local.virtual_network_resource_ids
   description = "A map of virtual network resource ids, keyed by the var.virtual_networks input map. Only populated if the virtualnetwork submodule is enabled."
 }
 
+output "resource_group_ids" {
```

please can you rename this to:

```suggestion
output "virtual_network_resource_group_ids" {
```

This means we can add other outputs for future submodules that may create RGs